workflow/pred_temperature_ridge.ipynb

```diff
@@ -0,0 +1,436 @@
+{
```

Comment:
<p>I add more information about <code>i_intervals</code> here. In the readme of lilio we have a more detailed explanation about how calendar works and the definition for anchor years and intervals. I also paste the link in this cell to make sure that user could find the information they need. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI4S2S/cookbook/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>